lib/tasks/import.rake

```diff
@@ -1,26 +1,23 @@
 # frozen_string_literal: true
 
-require "gds_api/json_client"
-
 namespace :import do
-  desc "Import all documents matching an organisation and document type from Whitehall Publisher, e.g. import:whitehall_migration[organisation-slug, document-type]"
+  desc "Import all documents matching an organisation and document type from Whitehall Publisher, e.g. import:whitehall_migration[\"cabinet-office\",\"NewsArticle\"]"
   task :whitehall_migration, %i[organisation_slug document_type] => :environment do |_, args|
-    WhitehallMigration.create!(organisation_slug: args.organisation_slug,
-                               document_type: args.document_type,
-                               start_time: Time.zone.now)
+    ActiveRecord::Base.transaction do
+      organisation_content_id = GdsApi.publishing_api.lookup_content_id(
+        base_path: "/government/organisations/#{args.organisation_slug}",
+      )
+
+      whitehall_migration = WhitehallImporter.create_migration(organisation_content_id, args.document_type)
```

Comment:
Shall we move the transaction into this method?

Reply:
Done